aws/resource_aws_macie2_classification_job_test.go

```diff
@@ -0,0 +1,534 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/hashicorp/aws-sdk-go-base/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/naming"
+)
+
+func testAccAwsMacie2ClassificationJob_basic(t *testing.T) {
+	var macie2Output macie2.DescribeClassificationJobOutput
+	resourceName := "aws_macie2_classification_job.test"
+	bucketName := "test-bucket-name-aws"
```

Comment:
Since S3 bucket names are unique across all of AWS, the bucket names need to be unique. You can use the following for each test

```suggestion
	bucketName := acctest.RandomWithPrefix("tf-acc-test")
```

You'll also need to import `"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"`